packages/components/src/components/contract-card/contract-card.scss

```
@@ -563,6 +563,12 @@
     }
     &__deal-cancel-fee {
         grid-area: deal-cancel-fee;
+        &__disabled {
+            color: $color-grey-6;
```

```suggestion
            color: var(--text-disabled-1);
```
To make it work for both light and dark theme

---

Done , thanks